src/GreaterThan.php

```diff
@@ -24,7 +24,7 @@ class GreaterThan extends AbstractValidator
      */
     protected $messageTemplates = [
         self::NOT_GREATER => "The input is not greater than '%min%'",
-        self::NOT_GREATER_INCLUSIVE => "The input is not greater or equal than '%min%'"
+        self::NOT_GREATER_INCLUSIVE => "The input is not greater or equal to '%min%'"
```

Comment:
Ideally, this should read `greater than or equal to`. I can make that change on merge.

Test failures are because a test was looking for an exact exception string.